multimodal/src/autogluon/multimodal/data/process_image.py

```diff
@@ -164,9 +164,12 @@ def __init__(
             else:
                 assert mmocr is not None, "Please install MMOCR by: pip install mmocr."
             cfg = model.model.cfg
-            cfg.data.test.pipeline = replace_ImageToTensor(cfg.data.test.pipeline)
-            self.val_processor = Compose(cfg.data.test.pipeline)
-            self.train_processor = Compose(cfg.data.test.pipeline)
+            try:  # yolov3
+                training_pipeline = cfg.data.train.dataset.pipeline
+            except:  # faster_rcnn
+                training_pipeline = cfg.data.train.pipeline
+            self.val_processor = Compose(replace_ImageToTensor(cfg.data.val.pipeline))
```

Comment:
What's the difference between `cfg.data.test.pipeline` and `cfg.data.val.pipeline`? The `val_processor` is used in both validation and testing, e.g., when calling `.predict()`, `.evaluate()`.

Reply:
In MMDet, the same pipeline is used for val and test in detection on COCO and VOC. So there is no difference.